docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1352,6 +1352,11 @@ Query Parameters:
 -   **memswap** - Total memory (memory + swap), `-1` to disable swap.
 -   **cpushares** - CPU shares (relative weight).
 -   **cpusetcpus** - CPUs in which to allow execution (e.g., `0-3`, `0,1`).
+-   **buildargs** – JSON map of string pairs for build-time variables. Users can
```

Comment:
Consistent referral to `buildargs`, tighten structure, expand link text for readability:

-->

-   **buildargs** – JSON map of string pairs for build-time variables. Users set
        these values in their environment. At build-time, Docker uses the `buildargs` as the environment context for command(s) run via the Dockerfile's `RUN` instruction or for variable expansion in other Dockerfile instructions. [Read more about the `buildargs` instruction](/reference/builder/#arg)